### DISCUSSION TITLE
Show close button only on the active tab

### DISCUSSION THREAD

**Discussion by pafke2** [piroor/treestyletab#3071] [Q&A] [answered]
Hello everyone, is there an option to hide (disable) close ('x') button on every tab but the active one? I know you can add some code lines to make it show on hover on tab or totally disable x button, but I couldn't find the option I mentioned above. I think it would be useful since you won't accidentally close any tab by missclicking or scrolling through tabs and still have the option to close a tab with a mouse without hitting 'ctrl + w' or whatever on your keyboard.

**Comment by piroor** [comment]
I think these snippets give us hints to do that.
https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#highlight-active-tab
https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#only-show-tab-close-button-on-hover-1448

**Comment by pafke2** [comment, marked as answer]
Thanks i managed to do this by adding
 ```
tab-item:not(.active) tab-closebox {
  display: none;
}
```

**Reply by piroor** [reply]
If you OK, please add that to the code snippets!
